index.html

```diff
@@ -11,16 +11,17 @@
     <link rel="stylesheet" href="./css/style.css">
     <title>Portfolio</title>
```

Comment:
- [ ] Please you page title does not meet accessibility requirements, the page title is not descriptive, please see the screenshot below and kindly fix that

![accessibility](https://user-images.githubusercontent.com/62849872/150979637-6d3f0c36-f11f-4bb0-9756-d99a80cc0d0d.png)